pkgs/applications/window-managers/wayfire/default.nix

```diff
@@ -1,33 +1,89 @@
-{ lib, stdenv, fetchurl, cmake, meson, ninja, pkg-config
-, cairo, doctest, libdrm, libexecinfo, libinput, libjpeg, libxkbcommon, wayland
-, wayland-protocols, wf-config, wlroots, mesa
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, meson
+, ninja
+, pkg-config
+, cairo
+, doctest
+, libdrm
+, libexecinfo
+, libinput
+, libjpeg
+, libxkbcommon
+, wayland
+, wayland-protocols
+, wayland-scanner
+, wlroots
+, mesa
+, pango
+, xorg
+, libevdev
+, libxml2
+, glm
+, useSystemWlroots ? true
```

Comment:
Why do we need this option?

Reply:
wlroots api is not stable，but for now, using system wlroots works fine